### PR TITLE
Output unit tests

### PR DESCRIPTION
**Pull request by iLigonis**
#output_unit_tests
Also added appsettings to gitignore

### CODE REVIEW

**Comment by mikebroughton** [comment]
I think it's worth fixing the conflicts before this PR is reviewed.

**Comment by mikebroughton** [comment]
Sigh. Didn't mean to close this.

**Comment by jogobom** [comment]
Some of this has been superseded by other PRs, so we definitely need to sort out the merge conflicts before reviewing.

File: YoCode/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿{
+  "duplicationCheckSetup": {
+    "CMDtoolsDir": "C:\\Users\\ukekar\\source\\repos\\Tools\\CMD"
+  }
+}
```
[the remark this answers is not in the document]
Comment (iLigonis) [reply YoCode/appsettings.json]: Ah, yea sorry, will remove it

**Pull request merged**